examples/nrf52840-edf/Cargo.toml

```diff
@@ -0,0 +1,21 @@
+[package]
+edition = "2021"
+name = "embassy-nrf52840-edf-examples"
+version = "0.1.0"
+license = "MIT OR Apache-2.0"
```

Comment:
Please add a publish = false here to prevent it from being picked up by embassy-devtool